platform/core/src/core.rs

```diff
@@ -117,7 +127,49 @@ pub fn load_feature_categories(
     )
 }
 
-pub fn exec_rdf_query(query: SPARQLQuery) -> Result<String, CoreFailure> {
+fn open_store(fs_root: String, active_feature_id: String) -> Result<Store, CoreFailure> {
+    Store::open(Path::new(&format!("{}/{}/{}", fs_root, active_feature_id, env!("RDF_DB_PATH")))).map_err(CoreFailure::map_storage_error)
+}
+
+pub fn open_feature_rdf_store() -> Result<(), CoreFailure> {
+    let mut instance = get_instance()?;
+    if instance.active_feature_id == None {
```

Comment:
`active_feature_id` is never set, there needs to be an API which the platform can call to set the currently `active_feature_id`.